### DISCUSSION TITLE
Leave backup in nested subfolder

### DISCUSSION THREAD

**Discussion by misakstvanu** [spatie/laravel-backup#1497] [General] [answered]
Hey! 
I ran into a problem with backup destination. Previously I didn't need to save backup to nested subfolders, but now when I do, I can not use "/" to navigate them. Am I missing something? If not, I would suggest to leave name sanitization to user. Thank you!

https://github.com/spatie/laravel-backup/blob/d50b31349e801e4590dcfa5a9db597f709431aa7/src/BackupDestination/BackupDestination.php#L29

**Comment by freekmurze** [comment, marked as answer]
I'd accept a PR that will allow `/` again.

**Comment by misakstvanu** [comment]
You got it, boss
